### PR TITLE
fix: display object name in overview

### PR DESCRIPTION
**Pull request by artemmufazalov**

### CODE REVIEW

File: src/containers/Tenant/Schema/SchemaInfoViewer/SchemaInfoViewer.js [hunk 1]
```diff
@@ -151,36 +157,43 @@ class SchemaInfoViewer extends React.Component {
             );
         }
 
-        if ([tabletMetricsInfo, partitionConfigInfo, tableStatsInfo.flat()].flat().length === 0) {
-            return <div className={b('item')}>Empty</div>;
+        if (
+            [generalTableInfo, tabletMetricsInfo, partitionConfigInfo, tableStatsInfo.flat()].flat()
+                .length === 0
+        ) {
```
Comment (ariser) [inline src/containers/Tenant/Schema/SchemaInfoViewer/SchemaInfoViewer.js:163]: I don't think this looks better than before. I understand this is what prettier does, but again, seems like we need to tune it. RN a solution might be to make it a variable

File: src/containers/Tenant/utils/schema.ts [hunk 2]
```diff
@@ -41,6 +43,49 @@ export const mapPathTypeToNavigationTreeType = (
...
+
+const databaseTypeToDBName: Record<ETenantType, string | undefined> = {
+    [ETenantType.UnknownTenantType]: 'Database',
+    [ETenantType.Domain]: 'Cluster Root',
+    [ETenantType.Dedicated]: 'Dedicated Database',
+    [ETenantType.Shared]: 'Shared Database',
+    [ETenantType.Serverless]: 'Serverless Database',
+};
+
+export const mapDatabaseTypeToDBName = (type?: ETenantType) => type && databaseTypeToDBName[type];
```
Comment (ariser) [inline src/containers/Tenant/utils/schema.ts:85]: what about i18n?

**Review by ariser — CHANGES_REQUESTED** [review]

File: src/containers/Tenant/utils/schema.ts (hunk 2 above)
Comment (artemmufazalov) [reply src/containers/Tenant/utils/schema.ts:85]: I don't really like the idea of translating special terms. Even if we decide to translate them, we should discuss correct naming together

Comment (ariser) [reply src/containers/Tenant/utils/schema.ts:85]: If they have translation in the docs, I believe they should be translated in UI. If not, then discussable. As for coming up with the naming — yes, of course we should decide with the team.

**Review by ariser — APPROVED** [review]

**Pull request merged**